core/flow/rule_manager.go

```diff
@@ -80,6 +80,15 @@ func GetRules() []*FlowRule {
 	return rulesFrom(tcMap)
 }
 
+func ClearRules() error {
+	isUpdated, err := LoadRules(nil)
+	if isUpdated {
+		return err
+	} else {
+		return errors.Wrap(err, "Fail to clear rules.")
```

Comment:
This does not indicate failure, as the rules may have been cleared before. Maybe here we could just return `err`?